src/Symfony/Component/Form/Extension/Core/Type/TelType.php

```diff
@@ -0,0 +1,33 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Form\Extension\Core\Type;
+
+use Symfony\Component\Form\AbstractType;
+
+class TelType extends AbstractType
```

Comment:
Hello everyone and thank you @apetitpa for this PR.

Reading the discussion here I understand everyone's point of view :).

>For what it's worth: the misd-service-development/phone-number-bundle (which integrates googlei18n/libphonenumber in Symfony) named it PhoneNumberType and I actually prefer it over Tel(ephone)Type, regardless of what is the html implementation behind.

We actually use this bundle in many applications, and its name sounds consistent to me as it handles phone numbers... with transformers, different widgets, etc.

Adding a simple type here is just a matter of matching the corresponding html input, there is no transformers to ensure the int type or whatever, so I would favor `TelType` as originally proposed by @apetitpa. I think it's consistent with other types that simply extend the `TextType` even if some may think that this is a "bad name" for a class.

I have no strong feeling about this however, if symfony deciders don't like it either I would then propose a simple `PhoneType`. What do you think?